testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/BaseTestHiveCoercion.java

```diff
@@ -225,6 +230,8 @@ protected void insertTableRows(String tableName, String floatToDoubleType)
                         "  DECIMAL '-12345678.123456123456', " +
                         "  '\uD83D\uDCB0\uD83D\uDCB0\uD83D\uDCB0', " +
                         "  '\uD83D\uDCB0\uD83D\uDCB0\uD83D\uDCB0', " +
+                        "  '2023-09-27', " +
+                        "  '1600-01-17', " +
```

Comment:
nit: Can we go for a negative year ?

Reply:
I don't know how to do that. The problem is that they are checked against types in this method https://github.com/trinodb/trino/blob/90315ecdb69cafeb757867699fd96750fe771a35/testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/BaseTestHiveCoercion.java#L378 and I need to put `java.sql.Date` there and it doesn't really support negative dates

Reply:
How about `0001-01-01` the first date